src/components/App.jsx

```diff
@@ -4,29 +4,55 @@ import Button from './Button'
 import Header from './Header'
 import Prompt from './Prompt'
 import prompts from '../data/prompts.json'
+import History from './History'
 
 function App() {
   const [currentPrompt, setCurrentPrompt] = useState(
     'Spin to receive your first prompt.'
   )
+  const [history, setHistory] = useState([])
 
   return (
     <Grid>
       <Header>Wheel of TBR</Header>
       <Main>
         <Prompt>{currentPrompt}</Prompt>
         <FlexWrapper>
-          <Button primary onClick={setRandomPrompt}>
+          <Button
+            disabled={currentPrompt.includes('Wheel is tired')}
+            primary
+            onClick={setRandomPrompt}
+          >
             Spin!
           </Button>
         </FlexWrapper>
+        {history.length ? <History history={history} /> : ''}
       </Main>
     </Grid>
   )
 
   function setRandomPrompt() {
-    const randomPrompt = prompts[Math.floor(Math.random() * prompts.length)]
-    setCurrentPrompt(randomPrompt)
+    function getRandomNumber() {
+      return Math.floor(Math.random() * prompts.length)
+    }
+    var randomPrompt = prompts[getRandomNumber()]
```

Comment:
Can't exactly tell you why, but you should use `let` instead of `var` in modern JS